### ISSUE TITLE
Color Picker Accessibility tasks

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#1627]
* No initial focus on any button when entering dialog box	
    * 2.4.3: Focus Order	
    * Would recommend placing initial focus on Default tab instead

* When initially tabbing in dialog box to reach Default no alt text  reads on Close button or reads “out of button button”	
    * 1.1.1: Alt text	
    * Should only read element of focus

* Inconsistent use of arrows to select hue base. 
    * Keyboard only: tab to range indicator and use arrow to move indicator  to desired color OR tab to indicator then arrow down to hue slider tool and  move with arrow keys 
With NVDA: tab to circle icon to hear instructions for slider. Then  arrow down to slider that reads “select view” then hit enter to activate the  slider, which is manipulated by any arrow key (screen reader only reads 1  number at a time in sequence as slider moves, i.e. 1, 2, 3, etc.)	
    * 2.1.1: Keyboard
    * 1.1.1: Alt text	
    * I’m not sure which use of arrows is intended for the hue slider

* Tabbing to Cancel button: reads “content info landmark” then “cancel  button”	
    * 1.1.1: Alt text	
    * Remove “content info landmark”

* Predefined colors: when tabbing to Default, the focus goes to the  second color, not transparent	
    * 2.4.3: Focus Order	
    * Move initial focus to transparent

* Predefined colors: transparent tab not described, reads: “list item 1  of 9”	
    * 1.1.1: Alt text	
    * Add “transparency grouping” (this is what is read under Custom) to  element

**Comment by interactivellama** [comment]
### Next steps
* Ask SLDS team about hue base and arrow keys on link with no role.
* Ask SLDS about buttons in footer
* Update text for transparent swatch to read transparent when it's empty string.
* If transparent is the selected color then it should be the selected working color when default swatches are shown. Currently black is the selected swatch, since it is the default working color, because transparent can't be the default working color since that would mean two transparent swatches on the Custom Color tab.

**Comment by interactivellama** [comment]
@davidlygagnon Here are the final a11y updates to Color picker.


## Engineering tasks
* Add updated description: "Saturation: 0%. Brightness: 11%" to anchor with role=[button] on the custom panel within an assistive text span. See below.
* Update text for transparent swatch to read "transparent" when it's empty string. We should add a `labels.transparent` string.
* Add `transparent` swatch to all examples and put it at the end. If transparent is the selected color, then it should be the selected working color when default swatches are shown.

![screen shot 2018-10-31 at 4 35 15 pm](https://user-images.githubusercontent.com/1290832/47822880-76b97d00-dd2b-11e8-8349-764a4ecfc5ef.png)

**Comment by interactivellama** [comment]
This will move Color Picker from prototype to production components.

**Comment by davidlygagnon** [comment]
Great, thanks for following up with SLDS.
